### ISSUE TITLE
Library symlink missing on Raspberry

### ISSUE THREAD

**Issue by pompushko** [mguentner/cannelloni#47]
Seems like library should be copied to **/lib** on Pi, not to **/usr/lib/**

```
pi@tag:~ $ ldd /usr/local/bin/cannelloni
        /usr/lib/arm-linux-gnueabihf/libarmmem-${PLATFORM}.so => /usr/lib/arm-linux-gnueabihf/libarmmem-v6l.so (0xb6f49000)
        libcannelloni-common.so.0 => not found
        libpthread.so.0 => /lib/arm-linux-gnueabihf/libpthread.so.0 (0xb6f1d000)
        libstdc++.so.6 => /lib/arm-linux-gnueabihf/libstdc++.so.6 (0xb6d95000)
        libgcc_s.so.1 => /lib/arm-linux-gnueabihf/libgcc_s.so.1 (0xb6d68000)
        libc.so.6 => /lib/arm-linux-gnueabihf/libc.so.6 (0xb6c14000)
        /lib/ld-linux-armhf.so.3 (0xb6f5c000)
        libm.so.6 => /lib/arm-linux-gnueabihf/libm.so.6 (0xb6ba5000)
```
After creating symlink to **/lib** - all works fine :)

thank you :)

**Comment by mguentner** [comment]
Which Linux distribution are you using? What is the content of your `LD_LIBRARY_PATH`?

**Comment by pompushko** [comment]
> Which Linux distribution are you using? What is the content of your `LD_LIBRARY_PATH`?

Debian Bullseye for Pi
LD_LIBRARY_PATH is empty.

**Comment by mguentner** [comment]
closed with #54
